src/Microsoft.AspNetCore.Mvc.Core/MvcOptions.cs

```diff
@@ -75,6 +75,11 @@ public MvcOptions()
         public FormatterCollection<IInputFormatter> InputFormatters { get; }
 
         /// <summary>
+        /// Gets or sets the flag to buffer the request body in formatters. Default is <c>true</c>.
+        /// </summary>
+        public bool BufferRequestBodyInFormatters { get; set; } = true;
```

Comment:
This should be named in such a way that it's `false` by default. It's very surprising when a flag isn't false by default.

Maybe `SupportInputFormatterBuffering`?